engine/src/main/resources/org/hibernate/validator/ValidationMessages.properties

```diff
@@ -13,7 +13,8 @@ javax.validation.constraints.Pattern.message     = must match "{regexp}"
 javax.validation.constraints.Size.message        = size must be between {min} and {max}
 
 org.hibernate.validator.constraints.CreditCardNumber.message        = invalid credit card number
-org.hibernate.validator.constraints.EAN.message                   = invalid {type} barcode
+org.hibernate.validator.constraints.Currency.message                = invalid currency (must be one of {value})
```

Comment:
Added, thanks. Waiting for Yoann's blessing on the French translation and it should be good.